### DISCUSSION TITLE
部署成功后无法注册用户Q: || Unable to register user after successful deployment Q:

### DISCUSSION THREAD

**Discussion by Code-wy** [walinejs/waline#2709] [Q&A] [answered]
### Describe the bug | 问题描述

数据库使用的是tidbcloud 服务端部署在vercel上，部署完成后尝试注册用户，请求200，vercel日志没有报错，但是注册用户没有成功
<img width="938" alt="微信图片_20240908192421" src="https://github.com/user-attachments/assets/c8a27f73-48f1-4965-b5e5-dd1c69856ab8">

### Website URL | 问题网站

https://waline.doraemon.press

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

TiDB

---
### Describe the bug | Problem description

The database uses tidbcloud and the server is deployed on vercel. After the deployment is completed, I try to register a user. The request is 200. The vercel log does not report an error, but the user registration is unsuccessful.
<img width="938" alt="WeChat picture_20240908192421" src="https://github.com/user-attachments/assets/c8a27f73-48f1-4965-b5e5-dd1c69856ab8">

### Website URL | Problem website

https://waline.doraemon.press

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

TiDB

**Comment by lizheming** [comment]
环境变量里是不是有 SERVER_URL ？看起来是变量没加 `https://` 开头的原因

---
Is there SERVER_URL in the environment variable? It seems that the variable does not start with `https://`

**Reply by Code-wy** [reply, marked as answer]
好的，可以了

---
Okay, that’s it
